app/api/client/logging.ts

```diff
@@ -0,0 +1,16 @@
+class Logger {
```

Comment:
Why not publish this logger as a npm package so that we can use it in other repos too? haha 🤣